CombatMetrics/ui/base/scroll_lists.lua

```
@@ -0,0 +1,393 @@
+		self.selections:Clear()
+	end
+end
+
+
+---@diagnostic disable: lowercase-global, unknown-symbol, exp-in-action, miss-symbol
+---@class DataEntryStruct
+---@field typeId number;
+---@field categoryId number;
+---@field data table;
+---@diagnostic disable-next-line: undefined-global
+hstructure DataEntryStruct
```

Luacheck (1.2.0)

[error] expected '=' near 'DataEntryStruct'

(E011)